src/integrationTest/java/uk/gov/hmcts/darts/testutils/PostgresIntegrationBase.java

```diff
@@ -30,13 +30,16 @@ public class PostgresIntegrationBase {
 
     @DynamicPropertySource
     static void configureProperties(DynamicPropertyRegistry registry) {
-
         registry.add("spring.datasource.url", POSTGRES::getJdbcUrl);
         registry.add("spring.datasource.username", POSTGRES::getUsername);
         registry.add("spring.datasource.password", POSTGRES::getPassword);
     }
 
     static {
+        // Keep an eye on this, if it needs to be increased as more tests utilise PostgresIntegrationBase then we may have a connection leak somewhere
+        final int serverMaxConnections = 50;
```

Comment:
This could be added to the integration test config or at least a final static of the class.

Reply:
Will declare as a class constant. I thought about adding it to the config yaml, but I decided against it as I don't want to encourage changes to it  - I think it shouldn't need to be changed unless we have a connection leak problem.